src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,12 @@
+$logo-size: 40px;
```

Comment:
You use it only once, so don't create variables for variables

Reply:
Check all your variables

Reply:
It's for mixin